libjiffy/bench/file_block_benchmark.cpp

```diff
@@ -122,5 +115,4 @@ int main(int argc, char const *argv[])
 	LOG(log_level::info) << "\t" << data_size << " payload";
 	LOG(log_level::info) << "\tThroughput: " << num_ops * 1E3 / tot_time << " requests per microsecond";
     
-    memkind_destroy_kind(pmem_kind);
```

Comment:
Where are all the destroy functions now?

Reply:
If we destroy kind before we free the space allocated by the kind, memory leak will be caused. I assume if the storage server exit, all the memory created on heap will be released.